waspc/data/Generator/templates/sdk/server/types/index.ts

```diff
@@ -0,0 +1,12 @@
+import { type Application } from 'express'
```

Comment:
This got moved from server do sdk. That is it really. I also updated one or two paths in server that used it.